polar-core/src/error.rs

```diff
@@ -77,6 +78,32 @@ impl PolarError {
             }
             _ => {}
         }
+
+        // Augment ParseSugar errors with relevant snippets of parsed Polar policy.
+        if let ErrorKind::Parse(ParseError::ParseSugar {
+            ref mut msg,
+            ref ranges,
+            ..
+        }) = self.kind
+        {
+            if let Some(source) = source {
+                match ranges.len() {
+                    1 => {
+                        let first = &source.src[ranges[0].0..ranges[0].1];
```

Comment:
Super minor nit: this would be a little easier to read if the range was stored as a `Range<usize>`, then it would be `&source.src[ranges[0]]`

Reply:
Yeah, definitely. Meant to do that and forgot. Good catch!